app/assets/javascripts/admin/property_toggle_ajax_handler.js

```diff
@@ -0,0 +1,88 @@
+function PropertyToggleAjaxHandler(options) {
+  this.actionsContainer = options.actionsContainer;
+  this.publishLinkAttribute = options.publishLinkAttribute;
+  this.unpublishLinkAttribute = options.unpublishLinkAttribute;
+  this.csrfToken = options.csrfToken;
+}
+
+PropertyToggleAjaxHandler.prototype.publishAjaxRequest = function(event) {
+  event.preventDefault();
+  _this = this;
+  $.ajax({
+    url: $(event.target).attr('data-publish-url'),
+    type: 'PUT',
+    data:  JSON.stringify({ id: $(event.target).attr('data-id') }),
```

Comment:
what if I simply write { id: $(event.target).attr('data-id') } w/o JSON.stringify? will it work?